### PR TITLE
Remove UNZIP and Git dependency from build step 1 & try to build step 1 in appveyor

### PR DESCRIPTION
**Pull request by ScalaWilliam**
Will also try to add appveyor thing here.

### CODE REVIEW

**Comment by lucasgautheron** [comment]
I am just curious, about replacing git archive with xcopy, how are .git and untracked files dealt with?

**Comment by ScalaWilliam** [comment]
They're not, I was thinking about why the file size of the .exe is 120MB :D

**Comment by ScalaWilliam** [comment]
we'll need another alternative anyway, the method should be one of selecting the files we want, rather than excluding them.

**Comment by ScalaWilliam** [comment]
we should never release from a local, stateful machine. The build *has* to be automated, fully.

**Comment by lucasgautheron** [comment]
The whole source directory can be excluded

**Comment by ScalaWilliam** [comment]
You do that in build step 2. Step 1 requires the sources.

File: source/vcpp/buildEnv/build_1_getfiles.bat [hunk 1]
```diff
@@ -1,12 +1,17 @@
+set tmp_ac=%TEMP%\af
+rd /s /q %tmp_ac%
+mkdir %tmp_ac%
+echo D | xcopy ..\..\.. %tmp_ac% /s /e /q
+rd /s /q %tmp_ac%\source\vcpp\buildEnv\ac
```
Comment (lucasgautheron) [inline source/vcpp/buildEnv/build_1_getfiles.bat]: Redundant with https://github.com/ActionFPS/ActionFPS-Game/blob/master/source/vcpp/buildEnv/build_2_prepare.bat#L21

**Comment by ScalaWilliam** [comment]
So it works - but runs the build twice. Which is acceptable right now. Will try NSIS soon. Might have an auto-building .exe soon :!!

**Pull request merged**